### PR TITLE
Documentation: Content Addition

### PR DESCRIPTION
**Pull request by TylerTrott**
**- What I did**
- Added content for snackbar demo app
- Minor changes for pr change requests

**- How I did it**
Local edits

**- How to verify it**
Review on devstaging

**- Description for the changelog**
Documentation: Content Addition

### CODE REVIEW

File: content/en/docs/Sample_Apps/arrive.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: "Arrive"
```
Comment (xavierchanth) [inline content/en/docs/Sample_Apps/arrive.md]: Same thing with @‎rrive vs Arrive, sorry missed this one last time.

**Review by xavierchanth — CHANGES_REQUESTED** [review]
Just the one comment, otherwise LGTM

File: content/en/docs/Sample_Apps/arrive.md (hunk 1 above)
Comment (TylerTrott) [reply content/en/docs/Sample_Apps/arrive.md]: :(

**Review by xavierchanth — APPROVED** [review]

**Review by TylerTrott — COMMENTED** [review]
Changes made for @ rrive

**Pull request merged**